examples/worldview/src/app/App.js

```diff
@@ -32,7 +32,8 @@ const KEPLER_UI = {
 import {MonitorPanel} from '../features/monitor/MonitorPanel';
 import {useKeplerDeckLayers, createSelectMapStyle} from '../features/kepler';
 
-import {useNewYorkScene} from '../scenes/newYork';
+// import {useScene} from '../scenes/newYork';
```

Comment:
What's the reason for name change from specific [city]Scene to general useScene?

Reply:
It's just easier to swap the imports if they have the same name.